### ISSUE TITLE
Total (only the value) above Stacked Bar Chart. It is possible to achieve?

### ISSUE THREAD

**Issue by Eduz6** [PhilJay/MPAndroidChart#4306]
Hello!

I implemented a Stacked Bar Chart. I have values and I divide the values as shown in the picture below:

![screenshot_20181113-160032](https://user-images.githubusercontent.com/45007503/48424239-83de5f00-e762-11e8-8519-1e950911fc6f.png)

```
switch (type) {
   case "1":
       yVals1.add(new BarEntry(i, new float [] {(float) time / 3, (float) time / 3, (float) time / 3}));
       break;
   case "2":
       yVals2.add(new BarEntry(i, new float [] {(float) time / 3, (float) time / 3, (float) time / 3}));
       break;
   case "3":
       yVals3.add(new BarEntry(i, new float [] {(float) time / 2, (float) time / 2}));
       break;
}
```

What I want to achieve is to see the total (the "time" variable) over each bar, like this:

![desire](https://user-images.githubusercontent.com/45007503/48425270-c2751900-e764-11e8-9df2-a97b1eb7ec79.png)

Is it possible to achieve this? If this is not possible, it would be great to create a new feature with this functionality!

Thank you so much in advance. Best regards!

**Comment by Eduz6** [comment]
I updated with:

`implementation 'com.github.philjay:mpandroidchart:-SNAPSHOT'`

I've tried the StackedValueFormatter that you implemented, but it doesn't solve my question.

What I want is put one label above each bar. Is that possible to achieve?

![desire new](https://user-images.githubusercontent.com/45007503/48617280-45c98100-e996-11e8-9dcc-1a7ae7f55b38.png)

Thank you for your attention.

**Comment by Eduz6** [comment]
**What I have:**
![screenshot_20181113-160032](https://user-images.githubusercontent.com/45007503/48841150-e18d2f80-ed90-11e8-8957-f60ca8be7b5d.png)

**What I want to achieve:**
![desire](https://user-images.githubusercontent.com/45007503/48841170-efdb4b80-ed90-11e8-8f30-91d572dcb8f4.png)

> two labels are being drawn for the **top-most bar**, one above the top line and one below the top line

This is just what I want to achieve! I want to put a label above each top-most bar, like a title! This is possible to achieve?

Thank you for your attention @almic

**Comment by Eduz6** [comment]
> **What I have:**
> ![screenshot_20181113-160032](https://user-images.githubusercontent.com/45007503/48841150-e18d2f80-ed90-11e8-8957-f60ca8be7b5d.png)
> 
> **What I want to achieve:**
> ![desire](https://user-images.githubusercontent.com/45007503/48841170-efdb4b80-ed90-11e8-8f30-91d572dcb8f4.png)
> 
> > two labels are being drawn for the **top-most bar**, one above the top line and one below the top line
> 
> This is just what I want to achieve! I want to put a label above each top-most bar, like a title! This is possible to achieve?
> 
> Thank you for your attention @almic

@almic, @PhilJay, or another admin, could you please answer my question? Is this possible to achieve with this library? Thank you and happy new year!

**Comment by ajaydhadhal1111** [comment]
Have you done this?

**Comment by Eduz6** [comment]
No... It seems that it's not possible to achieve with this library.

A sábado, 30/11/2019, 09:35, ajaydhadhal1111 <notifications@github.com>
escreveu:
